### PR TITLE
dm: add new config for lightning

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]
LGTM

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -106,7 +106,15 @@ loaders:                             # load 处理单元的运行配置参数
...
-
+    # 全量阶段数据导入的模式。可以设置为如下几种模式：
+    # - "sql"(默认)。使用 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) TiDB-backend 进行导入。
+    # - "loader"。使用 Loader 导入。此模式仅作为兼容模式保留，目前用于支持 TiDB Lightning 尚未包含的功能，预计在 6.0 版本完全废弃。
+    import-mode: "sql"
+    # 全量导入阶段针对冲突数据的解决方式：
+    # - "replace"（默认值）。仅支持 import-mode 为 "sql"，表示用最新数据替代已有数据。
+    # - "ignore"。仅支持 import-mode 为 "sql"，保留已有数据，忽略新数据。
+    # - "error"。仅支持 import-mode 为 "loader"。插入重复数据时报错并停止同步任务。
+    on-dupicate: "replace"
```
Comment (hfxsd) [inline dm/task-configuration-file-full.md:118]: 新增的2个参数是5.4的feature吗？

Comment (glorv) [reply dm/task-configuration-file-full.md:118]: 5.4 还没有，是下个版本的

Comment (sunzhaoyang) [reply dm/task-configuration-file-full.md:118]: 应该不是 6.0 完全废弃吧。这个配置加上就已经是 6.0 了

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: dm/task-configuration-file-full.md (hunk 1 above)
Comment (glorv) [reply dm/task-configuration-file-full.md:118]: 本来期望在 6.0 废弃，不过下个版本才发，应该还废弃不了，那可以改成预计会在后续版本废弃

**Comment by ran-huang** [comment]
@glorv Does this apply to v6.0?

**Comment by glorv** [comment]
> @glorv Does this apply to v6.0?

@ran-huang yes.

**Pull request merged**